packages/sanity/src/core/releases/store/createReleaseStore.ts

```diff
@@ -100,29 +136,18 @@ export function createReleaseStore(context: {
           resetOnSuccess: true,
         }),
         tap(() => fetchPending$.next(false)),
-        mergeMapArray((releaseSystemDocument: ReleaseSystemDocument) =>
-          previewStore
-            // Temporary release metadata document
-            .unstable_observeDocument(
-              `${RELEASE_METADATA_TMP_DOC_PATH}.${releaseSystemDocument.name}`,
-            )
-            .pipe(
-              map((metadataDocument) => ({
-                metadataDocument,
-                releaseSystemDocument,
-              })),
-            ),
-        ),
-        map((results) =>
-          results.flatMap(({metadataDocument, releaseSystemDocument}): ReleaseDocument[] => {
-            return metadataDocument && releaseSystemDocument
-              ? {...(metadataDocument as any), ...releaseSystemDocument}
-              : []
-          }),
+        map((releases) =>
+          releases.map(
+            (releaseDoc: ReleaseDocument): ReleaseDocument => ({
+              ...releaseDoc,
+              metadata: {...(releaseDoc as any).userMetadata, ...releaseDoc.metadata},
```

Comment:
userMetadata is going to be renamed to `metadata`  very soon – this makes sure both works